src/components/Company/CompanyName/CorrectNameToNumber.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <v-form lazy-validation>
+    <v-layout row>
+        <v-checkbox
+          v-model="correctToNumbered"
+          label="Change the company name to 1234567 B.C. Ltd."
```

Comment:
I know the mockup shows this, but maybe we should instead display "[Incorporation Number] B.C. Ltd."? Please check with Scott.

Reply:
Great points! I took the mock up literally hahah updated in next commit